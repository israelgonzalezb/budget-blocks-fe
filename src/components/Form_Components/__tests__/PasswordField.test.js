
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import PasswordField from '../PasswordField';


describe('PasswordField test', () => {
  it('PasswordField should match snapshot', () => {
    const component = renderer.create(<PasswordField
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
