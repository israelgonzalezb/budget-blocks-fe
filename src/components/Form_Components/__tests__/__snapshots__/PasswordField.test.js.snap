// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PasswordField test PasswordField should match snapshot 1`] = `
<div
  className="MuiFormControl-root"
>
  <p
    className="MuiTypography-root label MuiTypography-body1"
  >
     
     
  </p>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <div
      className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd"
        disabled={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="password"
      />
      <div
        className="MuiInputAdornment-root MuiInputAdornment-positionEnd"
      >
        <button
          aria-label="toggle password visibility"
          className="MuiButtonBase-root MuiIconButton-root MuiIconButton-edgeEnd"
          disabled={false}
          id="passwordVisibilityToggle"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 7c2.76 0 5 2.24 5 5 0 .65-.13 1.26-.36 1.83l2.92 2.92c1.51-1.26 2.7-2.89 3.43-4.75-1.73-4.39-6-7.5-11-7.5-1.4 0-2.74.25-3.98.7l2.16 2.16C10.74 7.13 11.35 7 12 7zM2 4.27l2.28 2.28.46.46C3.08 8.3 1.78 10.02 1 12c1.73 4.39 6 7.5 11 7.5 1.55 0 3.03-.3 4.38-.84l.42.42L19.73 22 21 20.73 3.27 3 2 4.27zM7.53 9.8l1.55 1.55c-.05.21-.08.43-.08.65 0 1.66 1.34 3 3 3 .22 0 .44-.03.65-.08l1.55 1.55c-.67.33-1.41.53-2.2.53-2.76 0-5-2.24-5-5 0-.79.2-1.53.53-2.2zm4.31-.78l3.15 3.15.02-.16c0-1.66-1.34-3-3-3l-.17.01z"
              />
            </svg>
          </span>
        </button>
      </div>
      <fieldset
        aria-hidden={true}
        className="PrivateNotchedOutline-root-97 MuiOutlinedInput-notchedOutline"
        style={
          Object {
            "paddingLeft": 8,
          }
        }
      >
        <legend
          className="PrivateNotchedOutline-legend-98"
          style={
            Object {
              "width": 0.01,
            }
          }
        >
          <span
            dangerouslySetInnerHTML={
              Object {
                "__html": "&#8203;",
              }
            }
          />
        </legend>
      </fieldset>
    </div>
  </div>
</div>
`;
